src/net/transport.rs

```diff
@@ -30,115 +36,134 @@ impl HttpTransport {
     #[must_use]
     #[allow(clippy::needless_pass_by_value)] // TODO: remove when ServerConfig is used
     pub fn new(
-        id: HelperIdentity,
+        identity: HelperIdentity,
         _server_conf: ServerConfig,
         network_conf: Arc<NetworkConfig>,
+        _callbacks: TransportCallbacks<Arc<Self>>,
     ) -> Arc<Self> {
+        #[cfg(never)]
```

Comment:
What is the story here?  Lots of cfg(never) chunks.

Reply:
These will be cleaned up in the next PR that restores the server.

(Once we've dealt with the known work items, we definitely need to do a pass to scrub any leftover `cfg(never)`s.